crates/core/tedge_actors/src/test_helpers.rs

```diff
@@ -309,7 +312,7 @@ where
 /// returning None, when no message is received after a given duration.
 pub struct TimedBoxReceiver<T> {
     timeout: Duration,
-    inner: T,
+    pub inner: T,
```

Comment:
Why is this required?

Reply:
> Needed this to access `send` messages using the message box.

Okay. I added `impl AsMut<T> for TimedBoxReceiver<T>` for that purpose. But `pub` works too.

> I can implement a `send()` method for this `TimedBoxReceiver` as well and delegate that to this `inner`.

Yes, that would be better than both `pub` and `AsRef`. You will need to add a constraint that `T` is a `Sender` ... and to `impl Sender for SimpleMessageBox`.